config.dev.toml

```diff
@@ -43,6 +43,14 @@ LogLevel = "debug"
 # Env override: ATHENS_CLOUD_RUNTIME
 CloudRuntime = "none"
 
+# ActivateDebugHandler specifies if the pprof endpoints should be exposed.
+# Note that this option is not meant to be activated forever on a server
+# and should be desactivated once not needed.
```

Comment:
typo: r/desactivated/deactivated

Also, AFAIK it's totally okay to keep it activated forever because it does not affect performance or the program unless the user actually pings the `/pprof` endpoint. Right?

Reply:
Yes !
Regarding having it enabled forever, I don't think it's a good idea. I can be source of DoS attacks  since it loads a lot of stuff on call. I think it's better to state it so people are aware of it and enable it when needed or in particular envs.

Reply:
Sounds good :)